charms/openfga-k8s/tests/charms/openfga_requires/src/charm.py

```diff
@@ -88,7 +88,7 @@ def _on_openfga_store_created(self, event: OpenFGAStoreCreateEvent):
             logger.info("secret content {}".format(content))
             self._state.token_secret_id = event.token_secret_id
         if event.token:
-            self._state.token = token
+            self._state.token = event.token
```

Comment:
Hmmm this is interesting, where was token coming from before?

Reply:
:cloud: ?